### PR TITLE
[Tests] add stop loss tests

### PR DESCRIPTION
**Pull request by GuillaumeDSM**
requires https://github.com/Drakkar-Software/OctoBot-Tentacles/pull/1396

### CODE REVIEW

File: additional_tests/exchanges_tests/abstract_authenticated_exchange_tester.py [hunk 1]
```diff
@@ -455,13 +455,23 @@ async def inner_test_create_and_cancel_stop_orders(self):
...
-                                                             trading_enums.TradeOrderSide.SELL)
-        self.check_created_stop_order(stop_loss, price, size, trading_enums.TradeOrderSide.SELL)
-        stop_loss_from_get_order = await self.get_order(stop_loss.exchange_order_id, stop_loss.symbol)
-        self.check_created_stop_order(stop_loss_from_get_order, price, size, trading_enums.TradeOrderSide.SELL)
-        assert await self.order_in_open_orders(open_orders, stop_loss)
-        await self.cancel_order(stop_loss)
+        assert self.exchange_manager.exchange.is_supported_order_type(
+            trading_enums.TraderOrderType.STOP_LOSS
+        ) is True
+        stop_loss = await self.create_market_stop_loss_order(
```
Comment (Herklos) [inline additional_tests/exchanges_tests/abstract_authenticated_exchange_tester.py:461]: :+1:

**Review by Herklos — APPROVED** [review]
